examples/flask/hello.py

```diff
@@ -19,10 +19,19 @@ def should_skip(app, environ):
     return "health/probe" in environ.get('PATH_INFO', '')
 
 def get_metadata(app, environ):
-    return {
-        'datacenter': 'westus',
-        'deployment_version': 'v1.2.3',
-    }
+    metadata = None
+
+    try:
+        metadata = {
+            'response-body': environ['response-body'],
+            'Content-Type': environ['response-headers']['Content-Type'.lower()],
```

Comment:
does the key have to be in the lowercase?

Reply:
removed